src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -91,9 +96,8 @@ class TezosNodeOperator(val node: TezosRPCInterface, batchConf: BatchFetchConfig
             val accountTry = Try(fromJson[Account](json)).map((id, _))
             accountTry.failed.foreach(_ => logger.error("Failed to convert json to an Account for id {}. The content was {}.", id, json))
             accountTry.toOption
-        }.flatten.toMap
-    )
-
+        }.flatten.toMap)
+    .map(_.mapValues(it => it.copy(script = it.script.map(toMichelsonScript[MichelsonCode]))))
```

Comment:
I'd suggest integrating this step in the internal `collect` process.

It would read better, especially removing the need to do nested mapping "after the fact"